### PR TITLE
Reorder playlists

### PR DESCRIPTION
**Pull request by ghost**
This reorders the playlist based on screenshots from the clients from cncnet, see https://imgur.com/a/zTXX0X7. I also made the track `credits` in RA hidden. I'm not sure about the order of the extensions but think it is less important than the order of the base game tracks. I haven't double checked this yet.

### CODE REVIEW

File: mods/cnc/audio/music.yaml [hunk 1]
```diff
@@ -1,103 +1,63 @@
...
 	VolumeModifier: 0.9
-ccthang: C&C Thang
+j1: Untamed Land (Jungle)
 	VolumeModifier: 0.7
-ind2: Canyon Chase (Industrial 2)
+jdi_v2: Just Do It Up 2 (Take 'Em Out)
+	VolumeModifier: 0.65
+radio: Radio
 	VolumeModifier: 0.7
-heavyg: Demolition (Heavy Gear)
```
Comment (reaperrr) [inline mods/cnc/audio/music.yaml]: You forgot to re-add this, should be the first track!

**Comment by reaperrr** [comment]
> I also made the track credits in RA hidden.

In that case you should make `Airstrike` and `I Am - Times` in TD hidden as well, as those were the credits tracks for GDI and Nod respectively, and not part of the track list in the original.

**Comment by matjaeck** [comment]
The TD list is probably not accurate. Youtube vidoes from original missions start with 
´Act on Instinct` so I will reorder it based on the order from https://cnc.fandom.com/wiki/Command_%26_Conquer_(1995)_soundtrack and then double check that nothing got lost.

**Comment by reaperrr** [comment]
Unlike RA, TD did not start with the first track in the track list, but rather the first track in the mix file (hence why it starts with AOI in the original without CovOps/patch1.06, despite the fact that it's the 6th track in the original).
If you used the CovOps CD, it would start with Depth Charge instead (CovOps tracks were all put before the original tracks in the mix file).

The screenshots are with Nyerguds' inofficial 1.06c patch applied, with some extra tracks enabled. So the tracklist on the screenshots is accurate, except that
```
Airstrike
C&C 80s Mix
Die!!
Reaching Out
Heartbreak
Enemies To Be Feared
Destructible Times
Walkürenritt/Ride of the Valkyries
```
were not enabled in the original prior to the inofficial patch.
And IMO, if we hide the Credits them in RA, we should also hide the tracks listed above.

If you do that and set AOI as `StartingMusic` in TD, I think we're set (strictly speaking, starting with Depth Charge would be more accurate, but I think AOI is way more popular, as it was the starting track before CovOps and possibly is the starting music with 1.06c applied as well, regardless of CovOps/extra tracks).

File: mods/cnc/audio/music.yaml (hunk 1 above)
Comment (GraionDilach) [reply mods/cnc/audio/music.yaml]: He added it into CovOps. Still wrong though.

File: mods/cnc/audio/music.yaml [hunk 2]
```diff
@@ -1,103 +1,63 @@
...
-stopthem: We Will Stop Them (Deception)
-	VolumeModifier: 0.7
-deception: We Will Stop Them (SFX)
-	Filename: stopthem
+rout: Reaching Out
 	Extension: var
-	VolumeModifier: 0.7
-aoi2: Act On Instinct (SFX)
-	Filename: aoi
+heart: Heartbreak
```
Comment (GraionDilach) [inline mods/cnc/audio/music.yaml]: Heartbreak is an SFX track, shouldn't it be among them?

File: mods/cnc/audio/music.yaml [hunk 3]
```diff
@@ -108,3 +68,58 @@ recn226m: Recon
...
+	Hidden: true
+	VolumeModifier: 0.7
+win2: Great Shot! (SFX)
+	Filename: win1
+	Extension: var
+	Hidden: true
+nod_win1: Nod Win Theme
+	Hidden: true
+	VolumeModifier: 0.9
+score: Militant Force (Scores)
```
Comment (GraionDilach) [inline mods/cnc/audio/music.yaml]: This doesn't belong here. MF was a RA1 track.

File: mods/cnc/audio/music.yaml (hunk 2 above)
Comment (GraionDilach) [reply mods/cnc/audio/music.yaml]: Same applies to Reaching Out.

**Comment by matjaeck** [comment]
Regarding TD, I am very tempted to list the tracks in the order of the OST. I listened to the whole playlist in their order and it just sounds really good and the order fits. I don't know, but I have the feeling that the songs played in this order when I think back of playing it on PS1. The order in the screenshots might be technically accurate but the OST order is musically superior.

File: mods/cnc/audio/music.yaml (hunk 2 above)
Comment (matjaeck) [reply mods/cnc/audio/music.yaml]: Will fix when we agreed on an order, see my comment.

File: mods/cnc/audio/music.yaml (hunk 3 above)
Comment (matjaeck) [reply mods/cnc/audio/music.yaml]: I will remove this.

**Comment by matjaeck** [comment]
Closing to use my time for testing and might reopen after the release is out and all the high priority refactors / fixes are done.

**Pull request closed without merging**